packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -517,6 +536,16 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		this.cdRef.detectChanges();
 	}
 
+	/**
+	 * Component lifecycle hook
+	 */
+	public ngAfterContentInit(): void {
+		if (this.customRowContent) {
+			this.customRowTemplate = this.customRowContent.template;
+			console.log("--- dummyTemplate", this.customRowTemplate);
```

Comment:
Could you please remove this `console.log` ?

Reply:
done